### ISSUE TITLE
confused by Blank label and space between words

### ISSUE THREAD

**Issue by jxlijunhao** [SeanNaren/warp-ctc#35]
in CTCLoss,  0 label is reserved for blank label, is it the same meaning as the  space between words?

**Comment by Separius** [comment]
No, they are not, the blank can be used to separate repeated characters ("aa" vs "a a" => "a/blank/a" and "a/space/a")
